server/controllers/boards.js

```diff
@@ -0,0 +1,158 @@
+const asyncHandler = require("express-async-handler");
+const Board = require('../models/Board');
+const Card = require('../models/Card');
+const Column = require('../models/Column');
+
+exports.createBoard = asyncHandler(async (req, res) => {
+  const { title } = req.body;
+  if (title) {
+    try {
+      const inProgress = new Column({ title: 'In Progress' });
+      const completed = new Column({ title : 'Completed' });
+      
+      await inProgress.save();
+      await completed.save();
+
+      const board = new Board({ title, columns: [inProgress._id, completed._id]});
+      const doc = await board.save();
+
+      res.status(201).json({
+        success: {
+          boardId: doc._id,
+        }
```

Comment:
Just for front-end purposes, wouldn't it make more sense to send down the whole board as opposed to just the id?

Reply:
Yeah that makes more sense, I'll make that adjustment.